src/ensembl/src/content/app/browser/chromosome-navigator/ChromosomeNavigator.scss

```diff
@@ -0,0 +1,45 @@
+@import 'src/styles/common';
+
+.chromosomeNavigator {
+  svg {
+    width: 100%;
+    overflow: visible;
+  }
+
+  .stick {
+    height: 4px;
+    fill: $light-grey;
+    stroke: $grey;
+  }
+
+  .viewport {
+    fill: white;
+  }
+
+  .viewportBorder {
+    fill: none;
+    stroke: $dark-grey;
+  }
+
+  .focusPointer {
+    fill: #ff9800; // FIXME: variable?
```

Comment:
Is this going to be added as a variable to the colour palette variables we have in settings?

Reply:
Added orange to our colour palette